io.elementary.capnet-assist.yml

```diff
@@ -0,0 +1,27 @@
+app-id: io.elementary.capnet-assist
+
+runtime: io.elementary.Platform
+runtime-version: 'daily'
+sdk: io.elementary.Sdk
+
+command: io.elementary.capnet-assist
+
+finish-args:
+  - '--share=ipc'
+  - '--socket=fallback-x11'
+  - '--socket=wayland'
+  - '--share=network'
+
+modules:
+  - name: gcr
+    buildsystem: meson
+    sources:
+        - type: git
+          url: https://gitlab.gnome.org/GNOME/gcr.git
+          tag: '3.36'
```

Comment:
The Gnome SDK already include gcr, no need to build it here too.

Reply:
Hmm alright, when I didn't include it as a module I would consistently get warnings surrounding gcr when I ran tests. I will go back and see if it is caused by something else!